package.json

```diff
@@ -18,19 +18,20 @@
   "types": "lib/peg.d.ts",
   "bin": "bin/peggy",
   "scripts": {
-    "pretest": "npm run parser",
-    "test": "npm run test:bare",
-    "test:bare": "mocha --recursive test --exclude test/browser.stub.js --reporter min",
+    "clean": "rimraf -f build && rimraf -f browser && mkdir build && cd build && mkdir ts && mkdir rollup && mkdir terser && cd .. && mkdir browser",
+    "terser": "terser -c passes=2 -m --sequences 40 build/rollup/peg.umd.js > build/terser/peggy.umd.min.js && terser -c passes=2 -m --sequences 40 --module build/rollup/peg.esmodule.js > build/terser/peggy.esmodule.min.js",
```

Comment:
Does terser support a config file?

Reply:
yes, using `--config-file`.  it's idiomatic to not use it but i can switch

Reply:
Would it be okay if I postponed this until after the config lands?  [It looks like terser's config file support is so un-used that it's a little wonky](https://github.com/terser/terser/issues/829), and given the other stuff in flight, I don't want to block

I would open a new issue and address this promptly